src/views/Blocks/HomeBlocks.js

```diff
@@ -1,12 +1,14 @@
 import React from 'react'
 import { ControlledAppbar } from '../../components'
+import { ControlledGrid } from '../../components'
 
 
 export const HomeBlocks = () => {
 
     return (
         <>
             <ControlledAppbar/>
+            <ControlledGrid/>
```

Comment:
This is not the proper way to use it. if the component doesn't need for homeblocks just don't implement.